components/menu/test/menu.test.js

```diff
@@ -176,51 +161,47 @@ describe('d2l-menu', () => {
 		});
 
 		it('shows nested menu when opener is clicked', async() => {
-			setTimeout(() => elem.querySelector('#b1').click());
+			setTimeout(() => clickElem(elem.querySelector('#b1')));
 			await oneEvent(elem, 'd2l-hierarchical-view-show-complete');
 			expect(nestedMenu.isActive()).to.be.true;
 		});
 
 		it('sets focus to d2l-menu-item-return when nested menu is displayed', async() => {
-			setTimeout(() => elem.querySelector('#b1').click());
+			setTimeout(() => clickElem(elem.querySelector('#b1')));
 			await oneEvent(elem, 'd2l-hierarchical-view-show-complete');
-			let focused = (document.activeElement.tagName === 'D2L-MENU-ITEM-RETURN');
-			if (!focused) {
-				focused = (document.activeElement === nestedMenu);
-			}
-			expect(focused).to.be.true;
+			await waitUntil(() => {
+				return (document.activeElement.tagName === 'D2L-MENU-ITEM-RETURN') ||
+					(document.activeElement === nestedMenu);
+			}, 'Focus on return');
```

Comment:
Also flaky, and this is one that I'm still seeing occasional flake locally so will be spending more time on it.